qa/tasks/cephfs/test_mirroring.py

```diff
@@ -1351,7 +1351,7 @@ def test_cephfs_mirror_cancel_mirroring_and_readd(self):
         log.debug('reconfigure client auth caps')
         self.get_ceph_cmd_result(
             'auth', 'caps', "client.{0}".format(self.mount_b.client_id),
-                'mds', 'allow rw',
+                'mds', 'allow rwps',
```

Comment:
oh, this is a new change. Why is it required for this fix?

Reply:
@vshankar 
Not sure how it came there. I was confused about using `rwps` back then. Anyway, dropped it. Please check.